### ISSUE TITLE
Duplicate Key Error when Building

### ISSUE THREAD

**Issue by georgiastuart** [cast-org/figuration#184]
[I also asked this on StackOverflow](http://stackoverflow.com/questions/42099386/duplicate-key-error-in-sass). 

I'm trying to build the Figuration source files and am running into this error:

```
Running "sass:core" (sass) task
>> Error: Duplicate key 1_5 in map (0: x: 0, y: 0, 0_25: x: $spacer-x * .25,
y: $spacer-y * .25, 0_5: x: $spacer-x * .5, y: $spacer-y * .5, 1: x: $spacer-
x, y: $spacer-y, 1_5: x: $spacer-x * 1.5, y: $spacer-y * 1.5, 2: x: $spacer-x 
* 2, y: $spacer-y * 2).
>>         on line 138 of scss/_settings.scss
>> >> $spacers: (
>>    ----------^
```

However, in _settings.scss the offending map looks fine to me:

``` sass
$spacers: (
0: (
    x: 0,
    y: 0
),
0_25: (
    x: ($spacer-x * .25),
    y: ($spacer-y * .25)
),
0_5: (
    x: ($spacer-x * .5),
    y: ($spacer-y * .5)
),
1: (
    x: $spacer-x,
    y: $spacer-y
),
1_5: (
    x: ($spacer-x * 1.5),
    y: ($spacer-y * 1.5)
),
2: (
    x: ($spacer-x * 2),
    y: ($spacer-y * 2)
)
) !default;
```

Googling has only turned up issues for much older versions of SASS and claims that it's been fixed, so I'm at a loss.

`sass -v` confirms I'm using selective steve.

In an attempt to resolve the issue, I uninstalled all local node packages and gems and reinstalled. It didn't help.

Building on macOS Sierra 10.12.2.

**Comment by georgiastuart** [comment]
I solved the problem and was able to build by:

1. enclosing the keys in the trouble map with quotation marks
2. deleting the & on line 315 of _dropdown.scss

I don't know enough about Sass to feel comfortable submitting a pull request though :)

**Comment by mbrambilla** [comment]
Thank you for the report.

Could you specify which branch you are on?  Possibly on the `v3-dev` branch?

**Comment by georgiastuart** [comment]
No, I had the key problem on both the 2.0.0 direct source download from the
website and from the master release branch. The & problem was on the master
release branch but not the download from the website.

**Comment by mbrambilla** [comment]
Ah,  I think I understand now.

There is a problem with the `spacers` map in `_settings.css`.  (Fixed in the `v3-dev` branch with #185).
In your local copy, as you stated, put quotes around the map keys to fix the compile issue.

The other problem is due to starting on the v3 development before actually creating the `v3-dev` branch, so the `master` branch is not at the v2.0.0 release state.  This is where the & issue for the `_dropdown.scss` is from. **My fault for poor commit handling.**

For now, I would recommend using a clean version of the v2.0.0 source,  available at http://figuration.org/get-started/download/, with the 'Download source' item.  The npm v2.0.0 version will also be clean.  These release versions will not have the in-process clutter from the upcoming v3.

My sincere apologies for the confusion.

**Comment by mbrambilla** [comment]
Closing ,as this was fixed with the v3.0.0-alpha.1 release.
